lemur/plugins/lemur_aws/sts.py

```diff
@@ -7,15 +7,29 @@
 """
 import boto
 import boto.ec2.elb
-
+import os
 from flask import current_app
 
 
-def assume_service(account_number, service, region=None):
-    conn = boto.connect_sts()
+class Credentials(object):
+    access_key = os.environ.get('AWS_ACCESS_KEY_ID')
+    secret_key = os.environ.get('AWS_SECRET_ACCESS_KEY')
+    session_token = None
+
 
-    role = conn.assume_role('arn:aws:iam::{0}:role/{1}'.format(
-        account_number, current_app.config.get('LEMUR_INSTANCE_PROFILE', 'Lemur')), 'blah')
+class Role(object):
+    credentials = Credentials()
+
+
+def assume_service(account_number, service, region=None):
+    try:
```

Comment:
I wonder if it doesn't make sense to keep the assume role architecture intact here, you would just add s3 permissions to the 'Lemur' role. This way you could also have the ability to write to s3 buckets in other accounts for free.